src/storage/Scarb.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "alexandria_storage"
+version = "0.1.0"
+description = "Starknet storage utilities"
+homepage = "https://github.com/keep-starknet-strange/alexandria/tree/src/storage"
+
+[[target.starknet-contract]]
+
```

Comment:
```suggestion

```

I don't see any contracts here, nor I don't think you'll want to build ones from this package? And not making it `[lib]` makes this package unusable as a dependency: https://docs.swmansion.com/scarb/docs/reference/targets.html#library

Reply:
Yes, correct on the contracts part, removing the `[[target.starknet-contract]]` declaration.

Re: `[lib]` - shouldn't this be present in all `Scarb.toml` files in all modules then?

Reply:
> Re: `[lib]` - shouldn't this be present in all `Scarb.toml` files in all modules then?

`[lib]` is the default and is automatically "added" if you don't specify any target :)